frontend_tests/node_tests/message_store.js

```diff
@@ -340,6 +346,15 @@ run_test("message_id_change", () => {
 
     set_global("message_list", {});
     set_global("home_msg_list", {});
+    set_global("all_message_lists", () => [home_msg_list]);
+
+    // Added to pass single line coverage.
+    const operators = [{operator: "stream", operand: "foo"}];
+    const mld = new MessageListData({
+        muting_enabled: false,
+        filter: new Filter(operators),
+    });
+    stored_mld.add(mld);
```

Comment:
@showell can you help @ryanreh99 develop a good node test suite for this data structure?  Ideally, such a test suite would have the property that we can do things like "confirm that message editing results in the right data structure".